### PR TITLE
adding support of python 3 to tools\startup.py

### PR DESCRIPTION
**Pull request by ArtyomKaltovich**
Common, now is 2017 and you still using python2. Shame on you :)
I have created fork for migrating to python3. I will update it after every task, but if somebody can help my, please welcome.

### CODE REVIEW

**Comment by ArtyomKaltovich** [comment]
Check now, please.

File: naive_bayes/nb_author_id.py [hunk 1]
```diff
@@ -22,12 +26,27 @@
...
+    return accuracy
 
+print("{:5} : {:5} : {:5} | {:5}".format("fit", "pred", "acc", "total"))
 
-#########################################################
-### your code goes here ###
-
-
-#########################################################
+clf = GaussianNB()
```
Comment (DTasev) [inline naive_bayes/nb_author_id.py]: I think this should be removed from this PR as it is a solution to the NB

File: naive_bayes/nb_author_id.py [hunk 2]
```diff
@@ -22,12 +26,27 @@
 features_train, features_test, labels_train, labels_test = preprocess()
 
 
+def fit_and_predict(clf, features_train, features_test, labels_train, labels_test):
```
Comment (DTasev) [inline naive_bayes/nb_author_id.py]: This function was in `tools/mylib.py`, I suggest removing the `mylib.py` file as it can be used to solve the tasks!

**Comment by ArtyomKaltovich** [comment]
Wow, so fast I've change my master branch. :)
I will create new branch and PR

**Pull request closed without merging**